packages/nx/src/utils/path.ts

```diff
@@ -5,14 +5,16 @@ function removeWindowsDriveLetter(osSpecificPath: string): string {
 }
 
 /**
- * Coverts an os specific path to a unix style path
+ * Coverts an os specific path to a unix style path. Use this when writing paths to config files.
+ * This should not be used to read files on disk because of the removal of Windows drive letters.
```

Comment:
This messaging will show up on nx.dev as well as intellisense for devkit users, I think its probably fine as-is, but maybe it'd be worth phrasing it more in line with it being ok for relative paths but not absolute? I can't think of a time that it wouldn't work when given relative path args.